Framework/PythonInterface/plugins/algorithms/SNSPowderReduction.py

```diff
@@ -215,8 +216,15 @@ def PyInit(self):
                              "List of all output file types. Allowed values are 'fullprof', 'gsas', 'nexus', "
                              "'pdfgetn', and 'topas'")
         self.declareProperty("OutputFilePrefix", "", "Overrides the default filename for the output file (Optional).")
-        self.declareProperty(FileProperty(name="OutputDirectory",defaultValue="",action=FileAction.Directory))
-        self.copyProperties('AlignAndFocusPowderFromFiles', 'CacheDir')
+        self.declareProperty(FileProperty(name="OutputDirectory", defaultValue="",action=FileAction.Directory))
+
+        # Optimization options
+        self.declareProperty(FileProperty(name='CacheDir', defaultValue='', action=FileAction.Directory))
+        self.declareProperty('CleanCache', False, 'Remove all cache files within CacheDir')
+        self.setPropertySettings('CleanCache', EnabledWhenProperty('CacheDir', PropertyCriterion.IsNotDefault))
+        property_names = ('CacheDir', 'CleanCache')
+        [self.setPropertyGroup(name, 'Optimization') for name in property_names]
```

Comment:
Can we change the group name from "Optimization" to "Caching"?  It almost sounds like we are trying to use some numerical methods here :sweat_smile:

Reply:
fixed